agent/lib/kontena/workers/fluentd_worker.rb

```diff
@@ -20,7 +20,7 @@ def initialize(autostart = true)
     end
 
     def start
-      observe(Actor[:node_info_worker]) do |node|
+      observe(Actor[:node_info_worker].observable) do |node|
```

Comment:
I can fix this back to `observe(Actor[:node_info_worker])` by adding a `Kontena::Observable::Helper` mixin module that provides an `add_observer` method on the `NodeInfoWorker` actor... however, then that needs some "magic" to be reintroduced into the `add_observer` method, such that it returns an `Kontena::Observable::Message` with  real `observable: self` value used to identify future observable updates, because the observable that the `Kontena::Observer.observe` gets is not the actual observable.

Design choice: Yes / No?

Reply:
Side-effect of that `Kontena::Observable::Helper#add_observer` proxying: the `observable.add_observer` loop would no longer actually be atomic, but the use of `Celluloid.exclusive { ... }` around it should still make it safe against lost messages for `Celluloid.receive`.

Alternative: add a `Kontena::Observable#observable` method that `return self`, and then `observables = observables.map{|o| o.observable }` to do the sync calls before the `add_observer` loop... that's just even more magic?

Reply:
> ... might be a bit confusing as one cannot observe any random actor that's not an `Observable`, right?

Pedantic: no actor can be an `Observable` anymore, but the `Kontena::Observable::Helper' could act as some kind of "Observable proxy".